index.html

```diff
@@ -180,6 +180,23 @@ <h3>Skills</h3>
             </ul>
         </section>
     </section>
+    <section class="last">
+        <div class="footer">
+        <h1 class="contact-title">Contact me</h1>
+        <p class="contact-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <br>
+        <form class="contact-form">
```

Comment:
- [ ] Form should use the action and method attribute to integrate with formspree service.
- action value should be similar to https://formspree.io/f/{form_id}
- method value equals POST
> please follow the link https://formspree.io/html, to learn more about the formspree and use it in your project